runtimes/pythonrt/py-sdk/autokitteh/events.py

```diff
@@ -18,8 +18,12 @@ def unsubscribe(subscription_id: str) -> None:
     pass
 
 
-def next_event(subscription_id: str, *, timeout: timedelta = None) -> AttrDict:
-    """Get the next event from the subscription."""
+def next_event(subscription_id: str | list, *, timeout: timedelta = None) -> AttrDict:
```

Comment:
```suggestion
def next_event(subscription_id: str | list[str], *, timeout: timedelta = None) -> AttrDict:
```

Reply:
the `| None` is not needed, it's automatic because of `= None`.